package/com.unity.formats.usd/Documentation~/USD Export.md

```diff
@@ -0,0 +1,77 @@
+# Exporting USD files
```

Comment:
we should note that Unity GameObjects set with "EditorOnly" tag will be ignored for export